src/selectors/per-thread/index.js

```diff
@@ -50,12 +55,17 @@ export const getThreadSelectors = (
   threadIndex: ThreadIndex
 ): ThreadSelectors => {
   if (!(threadIndex in _threadSelectorsCache)) {
-    const threadSelectors = getThreadSelectorsPerThread(threadIndex);
-    _threadSelectorsCache[threadIndex] = {
-      ...threadSelectors,
-      ...getStackAndSampleSelectorsPerThread(threadSelectors),
-      ...getMarkerSelectorsPerThread(threadSelectors),
+    const basicThreadSelectors = getThreadSelectorsPerThread(threadIndex);
```

Comment:
I'm finding all the naming of variables to be a bit overwhelming. Could we change the implementation to something like the following?

```js
// Create the selectors, and inject dependencies as needed.
let selectors = getThreadSelectorsPerThread(threadIndex);
selectors = {
  ...selectors,
  ...getStackAndSampleSelectorsPerThread(selectors),
  ...getMarkerSelectorsPerThread(selectors),
};
_threadSelectorsCache[threadIndex] = {
 ...selectors,
 ...getComposedSelectorsPerThread(selectors),
};
```

Reply:
ah yeah, why not reusing names, I lost the habit of doing that :)